diff --git a/src/settings/default.hpp b/src/settings/default.hpp
index eb31128..91c0f79 100644
--- a/src/settings/default.hpp
+++ b/src/settings/default.hpp
@@ -9,8 +9,8 @@ static const std::string defaultSettings =
     "\n"
 #if defined(WINE_RUNNER)
     "[wine]\n"
-    "dist = \"\"\n"
-    "type = \"wine64\"\n"
+    "dist = \"/app/proton\"\n"
+    "type = \"proton\"\n"
     "\n"
     "[wine.env]\n"
     "\"WINEDLLOVERRIDES\" = \"winemenubuilder.exe=d\"\n"
@@ -25,7 +25,7 @@ static const std::string defaultSettings =
     "[player.fflags]\n"
     "\n"
     "[player.env]\n"
-    "\n"
+    "SteamAppId = \"robloxplayer\"\n"
     "\n"
     "[studio]\n"
     "channel = \"\"\n"
@@ -36,4 +36,4 @@ static const std::string defaultSettings =
     "[studio.fflags]\n"
     "\n"
     "[studio.env]\n"
-    "\n";
+    "SteamAppId = \"robloxstudio\"\n";
